src/detect-bytetest.c

```diff
@@ -1279,6 +1369,59 @@ static int DetectBytetestTestParse22(void)
     return result;
 }
 
+/**
+ * \test Test bitmask option.
+ */
+static int DetectBytetestTestParse23(void)
+{
+    int result;
+    DetectBytetestData *data = NULL;
+    data = DetectBytetestParse("4, <, 5, 0, bitmask 0xf8", NULL, NULL);
```

Comment:
can we have FAIL_IF/FAIL_IF_NOT per line here, instead of the big `&&` chain?

Reply:
Good suggestion; will update.